contracts/utils/Whitelist.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
+
+import "./AccessControl.sol";
+
+contract Whitelist is AccessControl {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    event WhitelistEnabled(address sender);
+    event WhitelistDisabled(address sender);
+    event WhitelistAccountAdded(address account);
+    event WhitelistAccountRemoved(address account);
+
+    bytes32 public constant WHITELIST_ROLE = keccak256("WHITELIST_ROLE");
+
+    bool private _isWhitelistEnabled;
+    EnumerableSet.AddressSet private _whitelist;
+
+    modifier onlyWhitelistOrAdmin() {
+        _onlyWhitelistOrAdmin();
+        _;
+    }
+
+    modifier usingWhitelist() {
+        _usingWhitelist();
+        _;
+    }
+
+    function enableWhitelist() external onlyWhitelistOrAdmin {
+        require(!isWhitelistEnabled(), "Whitelist is already enabled");
+        _isWhitelistEnabled = true;
+        emit WhitelistEnabled(msg.sender);
+    }
+
+    function disableWhitelist() external onlyWhitelistOrAdmin {
```

Comment:
Add `renounceWhitelister` function. Its responsibility is to change whitelister address. Take a look to the [renounceAdmin](https://github.com/Cerebellum-Network/chainbridge-solidity/blob/master/contracts/Bridge.sol#L146) as an example. It can be executed by current whitelister or Admin.

Reply:
As we discussed, we're going to skip `renounceWhitelister` for now